frontend/lib/create_listing/view/create_listing.view.dart

```diff
@@ -0,0 +1,414 @@
+            border: OutlineInputBorder(
+              borderRadius: BorderRadius.circular(12),
+              borderSide: BorderSide.none,
+            ),
+            errorBorder: OutlineInputBorder(
+              borderRadius: BorderRadius.circular(12),
+              borderSide: const BorderSide(color: Color(0xFFFF5252), width: 1),
+            ),
+            focusedErrorBorder: OutlineInputBorder(
+              borderRadius: BorderRadius.circular(12),
+              borderSide: const BorderSide(color: Color(0xFFFF5252), width: 1),
+            ),
+            contentPadding: const EdgeInsets.symmetric(
+              horizontal: 16,
+              vertical: 14,
+            ),
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _buildConditionSelector() {
+    return Consumer<CreateListingViewModel>(
+      builder: (_, model, __) {
+        final bool showError = _showValidationErrors && model.condition.isEmpty;
+        
+        return Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            const Text(
+              'Condition',
+              style: TextStyle(
+                fontSize: 16,
+                fontWeight: FontWeight.w500,
+              ),
+            ),
+            const SizedBox(height: 8),
+            Row(
+              children: [
+                _buildConditionChip('New', model),
+                _buildConditionChip('Almost New', model),
+                _buildConditionChip('Used', model),
+                _buildConditionChip('Fair', model),
+              ],
+            ),
+            if (showError)
+              const Padding(
+                padding: EdgeInsets.only(top: 4),
+                child: Text(
+                  'Select a condition to continue.',
+                  style: TextStyle(
+                    color: Color(0xFFFF5252),
+                    fontSize: 12,
+                  ),
+                ),
+              ),
+          ],
+        );
+      },
+    );
+  }
+
+  Widget _buildConditionChip(String condition, CreateListingViewModel model) {
+    final isSelected = model.condition == condition;
+    return Padding(
+      padding: const EdgeInsets.only(right: 8),
+      child: ChoiceChip(
+        label: Text(
+          condition,
+          style: TextStyle(
+            color: isSelected ? Colors.black : Colors.white,
+          ),
+        ),
+        selected: isSelected,
+        selectedColor: Colors.grey[200],
+        backgroundColor: const Color(0xFF2C2C2C),
+        onSelected: (selected) {
+          if (selected) {
+            model.setCondition(condition);
+          }
+        },
+      ),
+    );
+  }
+
+  Widget _buildDescriptionField() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        TextFormField(
+          controller: _descriptionController,
+          maxLines: null,
+          minLines: 5,
+          validator: (value) {
```

Comment:
This description field can probably be merged together with the general "text input field", the validators can simply be passed in as a callback once you create a widget out of this.